integration/internal/termtest/stripansi.go

```diff
@@ -2,12 +2,12 @@ package termtest // import "github.com/docker/docker/integration/internal/termte
 
 import (
 	"errors"
-	"regexp"
 
 	"github.com/Azure/go-ansiterm"
+	"github.com/docker/docker/internal/lazyregexp"
 )
 
-var stripOSC = regexp.MustCompile(`\x1b\][^\x1b\a]*(\x1b\\|\a)`)
+var stripOSC = lazyregexp.New(`\x1b\][^\x1b\a]*(\x1b\\|\a)`)
```

Comment:
- This one won't be needed if the workaround is removed in https://github.com/moby/moby/pull/49195